src/System.Security.Cryptography.Xml/tests/EncryptionPropertyTest.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public class EncryptionPropertyTest
+    {
+        private const string validXml = @"<EncryptionProperty xmlns=""http://www.w3.org/2001/04/xmlenc#"" Id=""ID_val"" Target=""Target_val"" />";
+
+        [Fact]
+        public void DefaultConstructor()
+        {
+            EncryptionProperty encryptionProperty = new EncryptionProperty();
+            Assert.Null(encryptionProperty.Id);
+            Assert.Null(encryptionProperty.Target);
+            Assert.Null(encryptionProperty.PropertyElement);
+        }
+
+        [Fact]
+        public void ConstructorWithXmlElement_Valid()
+        {
+            XmlDocument doc = new XmlDocument();
+            doc.LoadXml(validXml);
+            EncryptionProperty encryptionProperty = new EncryptionProperty(doc.DocumentElement);
+            Assert.NotNull(encryptionProperty.PropertyElement);
+        }
+
+        [Fact]
+        public void ConstructorWithXmlElement_Element_Null()
+        {
+            Assert.Throws<ArgumentNullException>(() => new EncryptionProperty(null));
```

Comment:
Nit: for this or in the future, include the param name for exceptions for DerivedException: 
```
Assert.Throws<ArgumentNullException>("...", () => new EncryptionProperty(null));
```